### ISSUE TITLE
New way of hiding items from "creations" page

### ISSUE THREAD

**Issue by LeithBA** [hicetnunc2000/hicetnunc#285]
The burn functionality is finally here and it is great! :sparkles: Sadly it has issues.
 
 # Current Issues
1. It's not possible to burn an item we don't own anymore (e.g. a sold-out item). 
2. It's impossible to hide an item temporarily.
3. An object will still be hidden from the creators page if the person who bought it burns their copy (This will probably be fixed soon) #280 

# Potential Solution
This is a proposal to introduce a new way of hiding items from user pages without burning them.

The idea would be to introduce a new type of token file format (e.g. `filter.nunc`). This file would hold an array of all objkts IDs the user would like to hide from their profile. When loading a user page, we would need to check for the most recent `filter.nunc` file created by the user, parse it and remove all included objkts IDs from the creations list. 

To start, this file could be minted like any other token but it would not be hard to create a UI that generates the said file directly on hicetnunc.

This would allow the users to filter-out old obsolete objkts from their profiles even if they are sold-out and make them re-appear if needed.

# 
Going forward, this we could expand on the idea of a unique `.nunc` format to make more user custom configurations like how to sort creations etc...

#
👍  We can now vote on features. [Vote here if you like this](https://hicetnunc.featureupvote.com/suggestions/174544/a-new-way-to-hide-things-from-your-creations) 👍

**Comment by haustraliaer** [comment]
Ah thanks for writing this up - seems like this kind of thing could work well for variety of use cases... a kind of wallet "config" file - stored on the blockchain and able to be interpreted by multiple sources, not just hicetnunc.

Not sure what the implications are of "polluting" the raw minted objects, however... (or if that's even really a concern).

What are your thoughts on this @crzypatchwork ?

Is there a way we can "mint" this file without it actually being a "hicetnunc NFT"... an alternate minting contract perhaps (and even one that might be _mutable_ so one could add/remove items as they go)?

**Comment by andrevenancio** [comment]
`It's not possible to burn an item we don't own anymore (e.g. a sold-out item).`

If you dont own something, you can't burn it. regardless if you are the creator or not.

**Comment by haustraliaer** [comment]
This isn't about burning - but about hiding from your feed.

Think of it as tidying up, maybe you don't want to show the items that have sold out.

The owners would still have them in their collection.

**Comment by LeithBA** [comment]
`If you dont own something, you can't burn it. regardless if you are the creator or not.`

Yes! That's exactly my point @andrevenancio. Anything you've created and don't own anymore (sold out) is currently stuck on your profile forever. Burning should not be the only way to tidy up.

If burning is the "Delete" button, we should have an "Archive" button

**Comment by Quasimondo** [comment]
I like the idea of having "control tokens". This would be in the spirit that all data on hic et nunc is on-chain and not in some separate database. Maybe this might require a second smart contract - I think Rafael mentioned that possibility in one of the recent chats.

**Comment by LeithBA** [comment]
Yes! crzypatchwork confirmed this was in the making on [featureupvote](https://hicetnunc.featureupvote.com/suggestions/174544/a-new-way-to-hide-things-from-your-creations)
![image](https://user-images.githubusercontent.com/34186819/112817970-4fde0d00-9083-11eb-9047-07b4f580b165.png)

**Comment by dhowe** [comment]
see #588 for more discussion -- this is really an essential feature that has been missing for too long
